app/templates/route-components/buttons.hbs

```diff
@@ -51,8 +52,54 @@
 </section>
 
 <section>
+	<h2 id="primary">Dropdown button</h2>
+	<p>Adding <code class="format-code">.wds-dropdown</code> class and dropdown component inside button component will result in button with dropdown functionality</p>
```

Comment:
`button with dropdown functionality` or `button looking like dropdown`? There is no JS here, yet.